local_minio/minio.service

```diff
@@ -3,7 +3,7 @@ Description=Minio
 [Service]
 Environment=MINIO_ROOT_USER=access_key
 Environment=MINIO_ROOT_PASSWORD=secret_key
-ExecStart=/usr/bin/minio server  http://jimmy-minio-server-{8...15}/data/
+ExecStart=/usr/bin/minio server  http://minio-server-{0...#host_number#}/data/
```

Comment:
Let's write data to /data/$GROUPID and add the GROUPID as another variable here. That way we can have one specific location for the minio data and prevent conflicts with other apps